src/csvcubed/readers/cubeconfig/schema_versions.py

```diff
@@ -104,6 +115,11 @@ def _get_schema_version(
             QubeConfigJsonSchemaMajorVersion.v1,
             QubeConfigJsonSchemaMinorVersion.v2,
         )
+    elif schema_path == _v1_3_SCHEMA_URL:
+        return (
+            QubeConfigJsonSchemaMajorVersion.v1,
+            QubeConfigJsonSchemaMinorVersion.v3,
+        )
     else:
         raise ValueError(
             f"The $schema '{schema_path}' referenced in the cube config file is not recognised."
```

Comment:
Could you update this message to suggest the user checks for updates to csvcubed?